docker/util.sh

```diff
@@ -194,15 +211,17 @@ echo "Using GDAL_REPOSITORY=${GDAL_REPOSITORY}"
 IMAGE_NAME="${TARGET_IMAGE}-${TAG_NAME}"
 REPO_IMAGE_NAME="${DOCKER_REPO}/${IMAGE_NAME}"
 
+BUILD_ARGS=(
+    $DOCKER_CACHE_PARAM \
```

Comment:
it seems to me that DOCKER_CACHE_PARAM might not be initialized in the non-release case if neither --docker-cache nor --no-docker-cache have been specified